### ISSUE TITLE
training loss is nan

### ISSUE THREAD

**Issue by WoBuChiTang** [jasonppy/VoiceCraft#81]
Hello, I am training Chinese data with a weight of 830m. At the beginning, the loss is normal, but after 2000 steps, there is a situation of "loss is nan" in a large area. May I ask if you have encountered it when training gigaspeech?

**Comment by jasonppy** [comment]
use AdamW with smaller learning rate

see finetuning section in readme

**Comment by WoBuChiTang** [comment]
> use AdamW with smaller learning rate
> 
> see finetuning section in readme

Thank you!

**Comment by Magauiya** [comment]
Hi @WoBuChiTang! Have you fixed that error? Did suggestions (reducing the learning rate and using AdamW) from Jason help? If yes, can you please share hyperparameters that you have used?
